src/test/java/codeu/controller/RegisterServletTest.java

```diff
@@ -4,13 +4,23 @@
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
+import org.mockito.ArgumentCaptor;
 import org.mockito.Mockito;
 
+
+import codeu.model.data.User;
+import codeu.model.store.basic.UserStore;
+
+import org.mindrot.jbcrypt.*;
+
 public class RegisterServletTest
 {
   private RegisterServlet registerServlet;
+  private LoginServlet loginServlet;
```

Comment:
I'm not sure why this is added. You don't seem to use it anywhere in your testing. Delete?

Reply:
done!